tests/integration/assessment/test_assessment.py

```diff
@@ -1,5 +1,100 @@
-import pytest
+import logging
 
+from databricks.labs.ucx.assessment.crawlers import (
+    AzureServicePrincipalCrawler,
+    ClustersCrawler,
+    JobsCrawler,
+    PipelinesCrawler,
+)
+from databricks.labs.ucx.framework.crawlers import StatementExecutionBackend
 
-def test_table_inventory(ws, make_catalog, make_schema):
-    pytest.skip("test is broken")
+logger = logging.getLogger(__name__)
+
+
+def test_pipeline_crawler(ws, make_pipeline, make_schema, env_or_skip):
+    warehouse_id = env_or_skip("TEST_DEFAULT_WAREHOUSE_ID")
+    logger.info("setting up fixtures")
+    created_pipeline = make_pipeline(spn_example=1)
+
+    inventory_schema = make_schema(catalog="hive_metastore")
+    _, inventory_schema = inventory_schema.split(".")
+    backend = StatementExecutionBackend(ws, warehouse_id)
+
+    pipeline_crawler = PipelinesCrawler(ws=ws, sbe=backend, schema=inventory_schema)
+    pipelines = pipeline_crawler.snapshot()
+    results = []
+    for pipeline in pipelines:
+        if pipeline.success != 0:
+            continue
+        if pipeline.pipeline_id == created_pipeline.pipeline_id:
+            results.append(pipeline)
+
+    assert len(results) >= 1
+    assert results[0].pipeline_id == created_pipeline.pipeline_id
+
+
+def test_cluster_crawler(
```

Comment:
please rebase on latest `main`, i've added `inventory_schema` and `sql_backend` fixtures, which remove the need for `warehouse_id = env_or_skip("TEST_DEFAULT_WAREHOUSE_ID")`. Do it for all added integration tests.